data/budget.js

```diff
@@ -4,6 +4,12 @@ const users = require("./users");
 const uuid = require('node-uuid');
 
 let exportedMethods = {
+    
+    getAllBudget() {

```

Comment:
I don't think this method would serve any purpose.
At best we would need budgets by userId to update after expenses.